src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.focus.cs

```diff
@@ -0,0 +1,168 @@
+﻿#nullable enable
```

Comment:
I'm assuming this is from WinUI?

Reply:
Yes, this is ported code, but from the `ScrollView` control. The `ScrollViewer` code is very different from ours and it would not be possible to align, at least for now...